Added Links functionality

LalitNM opened the pull request:
Changes
- removed addEmoji function from client.js(now in script.js).
- made a function processMsg() which proesses a message.

abhigyank commented on public/script.js:
```diff
@@ -131,22 +131,78 @@ function SidebarToggle() {
 }
+
+function processMsg(raw_msg) {
+
```
Why is this function not in client.js from where it is called?

LalitNM replied:
I added this function there(script.js), let me resolve the conflicts, then It will be fine.

LalitNM replied:
Done.

abhigyank replied:
I mean that shouldn't this function be in client.js? Since it's being called from there.

P.S. better way to fix conflicts with master is to use git rebase master instead of merge, this doesn't create "merge" commit and you have clean commit history.

LalitNM replied:
I was in an impression that we are keeping all these functions in `script.js`:thinking:

abhigyank replied:
Cool let's keep it here then.

abhigyank commented:
@LalitNM the resolving of links looks awesome and it will be cool to have that.
I feel we could use a library here to do that, it will make our lives much easier and we wouldn't create any bugs or corner case issues. 
 Check out [linkify](https://www.npmjs.com/package/linkifyjs) , it will do all of it with just one line of function call.

LalitNM commented:
> it will do all of it with just one line of function call.

I did bruteforce :tired_face: to find links :laughing:
Okay I will use that library now.

LalitNM committed:
Added links functionality

Changes:
- added linkifyjs in dependencies
- updated app.js to linkify text messages

LalitNM commented:
@abhigyank, is this fine now? Or it can be further improved?

LalitNM commented:
This looks good, merging this.
